workflows/MODEL_TO_DATA.nf

```diff
@@ -53,24 +53,49 @@ include { VALIDATE } from '../modules/validate_model_to_data.nf'
 include { SCORE_MODEL_TO_DATA as SCORE } from '../modules/score_model_to_data.nf'
 include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_VALIDATE } from '../modules/annotate_submission.nf'
 include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_SCORE } from '../modules/annotate_submission.nf'
+include { ANNOTATE_SUBMISSION as ANNOTATE_SUBMISSION_AFTER_UPDATE_FOLDERS } from '../modules/annotate_submission.nf'
 include { SEND_EMAIL } from '../modules/send_email.nf'
 
 workflow MODEL_TO_DATA {
+
+    // Phase 0: Each submission is evaluated in its own separate channel
```

Comment:
I like the comments dividing the workflow logically up into phases. You might consider looking at turning all of these into formal subworkflows in the future.